app/crowd/models.py

```diff
@@ -20,6 +20,13 @@ class Meta:  # noqa: D106
         verbose_name = _("user")
         verbose_name_plural = _("users")
         db_table = "cr_user"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["auth0_id"],
+                condition=~models.Q(auth0_id=""),
```

Comment:
Almost missed `~`, and it looked like unique empty string XD

Reply:
We want to ensure non-empty auth0 ids are unique